core/client/src/main/scala/org/http4s/otel4s/middleware/client/UriTemplateClassifier.scala

```diff
@@ -65,4 +65,12 @@ object UriTemplateClassifier {
 
   /** A classifier that does not classify any URI templates. */
   val indeterminate: UriTemplateClassifier = _ => None
+
+  /** Somewhat similar to `HttpRoutes.of` for use with `Http4sDsl`. */
```

Comment:
Extra credit: putting an example directly in the scaladoc here, to make that connection clear.

Reply:
I actually just hopped on to do that right now!